assets/dev/js/editor/elements/views/container.js

```diff
@@ -419,6 +426,29 @@ const ContainerView = BaseElementView.extend( {
 			this.ui.percentsTooltip.hide();
 		}
 	},
+
+	droppableDestroy( settings ) {
+		if ( settings.containerDroppable ) {
+			this.$el.html5Droppable( 'destroy' );
+		}
+
+		if ( settings.containerDroppableInner ) {
+			this.$el.find( '> .e-container__inner' ).html5Droppable( 'destroy' );
+		}
+
+		settings.containerDroppable = false;
+		settings.containerDroppableInner = false;
+	},
+
+	droppableInitialise( settings ) {
+		if ( 'boxed' === settings.get( 'content_width' ) ) {
+			this.$el.find( '> .e-container__inner' ).html5Droppable( this.getDroppableOptions() );
+			settings.containerDroppableInner = true;
+		} else {
+			this.$el.html5Droppable( this.getDroppableOptions() );
+			settings.containerDroppable = true;
+		}
```

Comment:
1. I found this aspect very complicated because we need to know this also after update the content_width settings. I got a lot of JS errors, because in some cases I destroyed the wrong version (container instead of container inner or the other way around). Therefore I stored this in two variables.

2. I am looking into a isActive() method now. I might need help with that. I will update Tzvi and also check if Evyatar is available. 

4. Done in next update.